Build-JellyFin.ps1

```diff
@@ -1,18 +1,25 @@
 ﻿[CmdletBinding()]
 param(
     [switch]$InstallFFMPEG,
+    [switch]$InstallNSSM,
     [switch]$GenerateZip,
-    [string]$InstallLocation = "$Env:AppData/JellyFin-Server/",
+    [string]$InstallLocation = "$Env:AppData/Jellyfin-Server/",
     [ValidateSet('Debug','Release')][string]$BuildType = 'Release',
     [ValidateSet('Quiet','Minimal', 'Normal')][string]$DotNetVerbosity = 'Minimal',
     [ValidateSet('win','win7', 'win8','win81','win10')][string]$WindowsVersion = 'win',
     [ValidateSet('x64','x86', 'arm', 'arm64')][string]$Architecture = 'x64'
 )
 
-if($IsWindows){
-    $TempDir = $env:Temp
+#Check if we are on PowershellCore or not.
+if($PSVersionTable.PSEdition -eq 'Core'){

```

Comment:
Cannot you do it like (pseudo code):

```
if is_pwsh_core and not is_windows:
    temp=mktemp -d
else:
    temp=env:temp
```

Reply:
If it helps readability sure, fixed in latest.